lab_5_scrapper/scrapper_config.json

```diff
@@ -1,9 +1,15 @@
 {
-    "seed_urls": [],
-    "headers": {},
-    "total_articles_to_find_and_parse": 0,
-    "encoding": "",
-    "timeout": 0,
+    "seed_urls": ["https://www.vremyan.ru/analitycs",
+        "https://www.vremyan.ru/analitycs?&page202=2",
+        "https://www.vremyan.ru/analitycs?&page202=3"],
+    "headers": {"cookie" : "sdata[VisitTime]=2024-04-08%2013%3A19%3A31; sdata[referer]=https%3A%2F%2Fyandex.ru%2F; twk_uuid_5c66e79a7f688621d571f5c1=%7B%22uuid%22%3A%221.2U66pnZJhhmKjpBsuZ4SjlSE4f1yGgaYneOfh5qC8LA8RgGH8Lo8Ue6ojb49x8ucF9Op2qOmLQXDsxnIED7H8XkDGSjckXklnYiqiWUKx1GkAJh6vJ4pYnY40clSrSA%22%2C%22version%22%3A3%2C%22domain%22%3A%22supermonitoring.com%22%2C%22ts%22%3A1712575206503%7D; ppms_privacy_5175571d-16e9-40c5-9eca-bcb6d5e49f20={%22visitorId%22:%22de0e8ba5-abac-4c2b-aaac-3fcf467ab7b1%22%2C%22domain%22:{%22normalized%22:%22www.supermonitoring.com%22%2C%22isWildcard%22:false%2C%22pattern%22:%22www.supermonitoring.com%22}%2C%22consents%22:{%22analytics%22:{%22status%22:-1}%2C%22user_feedback%22:{%22status%22:-1}%2C%22marketing_automation%22:{%22status%22:-1}}%2C%22staleCheckpoint%22:%222024-04-08T11:20:50.079Z%22}; stg_returning_visitor=Mon%2C%2008%20Apr%202024%2011:32:39%20GMT; stg_traffic_source_priority=1; _ga_KLCW8G3CY6=GS1.1.1713941532.1.0.1713941532.0.0.0; _ga=GA1.1.1683454393.1713941532; stg_last_interaction=Wed%2C%2024%20Apr%202024%2006:52:13%20GMT",
+    "accept-language" : "ru,en;q=0.9",
+        "user-agent" : "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 YaBrowser/24.1.0.0 Safari/537.36",
+    "accept" : "text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.7"
+    },
+    "total_articles_to_find_and_parse": 10,
```

Comment:
let's lower the number of articles for CI to pass quicklier as well 

e.g. to 3

Reply:
thank you) done